core/ui/DefaultSearchResultList.tid

```diff
@@ -5,11 +5,24 @@ caption: {{$:/language/Search/DefaultResults/Caption}}
 \define searchResultList()
 //<small>{{$:/language/Search/Matches/Title}}</small>//
 
-<$list filter="[!is[system]search:title{$(searchTiddler)$}sort[title]limit[250]]" template="$:/core/ui/ListItemTemplate"/>
+<$list filter="[!is[system]search:title{$(searchTiddler)$}sort[title]limit[250]]">
+<$list filter="[<currentTiddler>match{$:/temp/search}]" variable="ignore" emptyMessage="""
+{{||$:/core/ui/ListItemTemplate}}
+""">
+<span class="tc-list-item-selected">
+{{||$:/core/ui/ListItemTemplate}}
+</span>
+</$list></$list>
 
 //<small>{{$:/language/Search/Matches/All}}</small>//
 
-<$list filter="[!is[system]search{$(searchTiddler)$}sort[title]limit[250]]" template="$:/core/ui/ListItemTemplate"/>
+<$list filter="[!is[system]search{$(searchTiddler)$}sort[title]limit[250]]">
+<$list filter="[<currentTiddler>match{$:/temp/search}]" variable="ignore" emptyMessage="""
+{{||$:/core/ui/ListItemTemplate}}""">
+<span class="tc-list-item-selected">
```

Comment:
I think this might be done more easily as:

```
<$list filter="[!is[system]search:title{$(searchTiddler)$}sort[title]limit[250]]">
<span class={{{[<currentTiddler>match{$:/temp/search}then[tc-list-item-selected]] }}}>
<$transclude tiddler="$:/core/ui/ListItemTemplate"/>
</span>
</$list>
```

Reply:
thanks @Jermolene, I've applied your suggestion